doc/TBG_macros.cfg

```diff
@@ -198,6 +198,11 @@ TBG_liveViewYZ="--<species>_liveView.period 1 --<species>_liveView.slicePoint 0.
 # Print the maximum charge deviation between particles and div E to textfile 'chargeConservation.dat':
 TBG_chargeConservation="--chargeConservation.period 100"
 
+# ParticleDetector: detects particles at infinite distance
```

Comment:
should read: `(virtually) propagates and collects particles to infinite distance`

the question is: does this only work for massless particles? write photons in that case to not obfuscate the use case.